.eslintignore

```diff
@@ -1,3 +1,4 @@
+lib/internal/v8.js
```

Comment:
The file is whitelisted because the V8 builtins don't lint.  They're not legal JS syntax, eslint can't parse them.